### ISSUE TITLE
ERCOT API Now Requires Key

### ISSUE THREAD

**Issue by WillKoehrsen** [kmax12/gridstatus#339]
Using the ERCOT API now requires a key.

```python
from gridstatus.ercot_api.ercot_api import hit_ercot_api
import datetime
import pytz

two_days_ago = datetime.datetime.now(tz=pytz.UTC) - datetime.timedelta(days=2)
actual_by_wzn_endpoint = "/np6-345-cd/act_sys_load_by_wzn"
two_days_actual_by_wzn = hit_ercot_api(
    actual_by_wzn_endpoint, operatingDayFrom=two_days_ago
)

=> KeyError: 'fields'
```

If you look at a response to a request you can see the issue:

```python
 import requests

response = requests.get('https://api.ercot.com/api/public-reports/np6-345-cd/act_sys_load_by_wzn')
response.json()

=> {'statusCode': 401,
 'message': 'Access denied due to missing subscription key. Make sure to include subscription key when making requests to an API.'}
```

We need to update the codebase to support an ERCOT API key. This should be user-supplied. We will also need to figure out how to get an ERCOT API key into the test workflow.

**Comment by WillKoehrsen** [comment]
I spent about 10 minutes trying to figure out how to get an API key from ERCOT but couldn't figure it out. If someone with more patience wants to try, that would be much appreciated. 

The API still seems like it's in pre-release.

**Comment by WillKoehrsen** [comment]
Addressed by #348. 

* Register for an account [here](https://apiexplorer.ercot.com/)
* Follow directions [here](https://developer.ercot.com/applications/pubapi/ERCOT%20Public%20API%20Registration%20and%20Authentication/) to get a subscription key
* Set the `ERCOT_` env variables in a `.env` file (see `.env.template` for an example) or `export` the variables on the command line
